File: Semestru 3/MAP/practice_practic/fapte_bune/fapte_bune/src/domain/Entity.java
package domain;

public class Entity {
    private long ID;
    private long SerialVersionUID;

    public long GetID () {
        return this.ID;
    }

    public long GetSerialVersionUID() {
        return this.SerialVersionUID;
    }

    public void SetID(long id) {
        this.ID = id;
    }

    public void SetSerialVersionUid(long serialVersionUID) {
        this.SerialVersionUID = serialVersionUID;
    }
}

File: Semestru 4/MPP/lab/.net/lab1/lab1/Models/Domain/Entity.cs
namespace lab1.Models.Domain
{
	public class Entity<ID>
	{
		public ID Id { get; set; }

        public Entity()
        {
                
        }

        public Entity<ID> SetId (ID id)
		{
			this.Id = id;
			return this;
		}
	}
}
